### DISCUSSION TITLE
Where to store component state

### DISCUSSION THREAD

**Discussion by develmac** [vuejs/pinia#1422] [Help and Questions]
Dear Pinia community, 

I can remember from all the Redux discussions that component state typically should not be stored in a shared state store and should remain local to the component. Is this the same for Pinia? In most of the examples I can see that component state is pushed into Pinia. 

Are there any best practices documented for this?

**Comment by posva** [comment]
yes, it's the same. Component state should not go in the store, it should go in the component.

**Reply by develmac** [reply]
Thanks, that is a clear answer! My guides show though how the state of components is stored in the store. Maybe a clarification in the docs would be welcome?
